apps/web/screens/Layouts/Layouts.tsx

```diff
@@ -150,4 +150,32 @@ export const NewsItemLayout: FC<NewsItemProps> = ({ sidebar, children }) => (
   </GridContainer>
 )
 
+interface SubpageProps {
+  main: ReactNode
+  details?: ReactNode
+}
+
+export const SubpageLayout: FC<SubpageProps> = ({ main, details }) => {
+  return (
+    <Box width="full" paddingTop={10}>
+      <Box paddingBottom={6}>
+        <GridContainer>
+          <GridRow>
+            <GridColumn span="12/12">{main}</GridColumn>
+          </GridRow>
```

Comment:
same as before, and here it might be enough to have one grid container wrapping the layout